src/mui/input/SelectArrayInput.js

```diff
@@ -130,9 +178,44 @@ export class SelectArrayInput extends Component {
         };
     }
 
+    renderCreateButton = () => (
+        <MenuItem
+            rightIcon={<AddIcon color={cyan500} />}
+        >
+            <span
+                style={{
```

Comment:
This create a new style object at every renders. Extract it outside: `style={styles.menuItem}`

See the **Beware of Object Literals in JSX** section in this [article](https://marmelab.com/blog/2017/02/06/react-is-slow-react-is-fast.html).